src/main/resources/application.yml

```
@@ -35,10 +35,24 @@ spring:
           time_zone: UTC
   jta:
     enabled: false
+  
+  redis:
+    host: 100.75.227.17  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)